playbook/L3/roles/l3-basic-functionality/tasks/l3-basic-functionality.yml

```diff
@@ -500,7 +500,7 @@
       var: result
     when: (inventory_hostname == "DUT") 
   - name: TC9 - Configure IPv4 address to bridge {{bridge}} on TR
-    command: bash -c "ip addr flush dev {{bridge}}; ip addr add {{IPAddr2}}/{{prefixlen}} dev {{bridge}}; ifconfig {{bridge}}"
+    command: bash -c "ip addr flush dev {{bridge}} && ip addr add {{IPAddr2}}/{{prefixlen}} dev {{bridge}} && ifconfig {{bridge}} && ping -c 3 {{IPAddr1}}"
```

Comment:
Make sure that there are proper handling when ping fails

Reply:
Once the ping fails (rc =1) test case should fail , rescue section and always section will be executed,  no need of extra code for ping failing handling.